src/apiary.apib

```diff
@@ -7,6 +7,8 @@ HOST: https://api.teamleader.eu
 
 :[OAuth2](oauth2.apib)
 
+:[Changelog](changelog.apib)
```

Comment:
Something to add to the bottom maybe?

As a developer I wouldn't want to scroll through a big changelog just to see the first endpoint...

Reply:
Not sure if we want to embed the entire changelog within the documentation. We will probably just link to a `CHANGELOG.md` file then.

Reply:
That's something I can live with 👍